tox.ini

```diff
@@ -1,25 +1,25 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py{39,310,311,312}-{linux,macos,windows}-pyqt, py{39,310}-{linux,macosintel,windows}-pyside
+envlist = py{310,311,312,313}-{linux,macos,windows}-pyqt5, py{310,311,312,313}-{linux,macos,windows}-pyqt6, py{310}-{linux,macos_x86_64,macos_arm64,windows}-pyside2, py{310}-{linux,macos_x86_64,macos_arm64,windows}-pyside6
 
 [gh-actions]
 python =
-    3.9: py39
     3.10: py310
     3.11: py311
     3.12: py312
+    3.13: py313
 
 [gh-actions:env]
 PLATFORM =
-    ubuntu-latest: linux
-    macos-latest: macos
-    macos-13: macosintel
-    windows-latest: windows
+    macos_arm64: darwin-arm64
+    macos_x86_64: darwin-x86_64
+    linux: linux
+    windows: win32
```

Comment:
I think this needs to be reverted, because this is supposed to map the GitHub runner (`platform`) to the tox env.
The runners are defined in test_and_deploy:
`platform: [ ubuntu-latest, windows-latest, macos-latest ]`
So looks like we dropped testing on macosintel and could just remove that.


Or it needs to be changed to have the GitHub names on the left hand side...